src/status_im2/subs/wallet/wallet.cljs

```diff
@@ -51,3 +53,29 @@
    (-> wallet
        (get-in [:accounts current-viewing-account-address])
        (assoc :balance (get balances current-viewing-account-address)))))
+
+(defn- calc-token-value
+  [{:keys [symbol market-values-per-currency] :as item}]
+  (let [fiat-value                        (utils/total-token-value-in-all-chains item)
```

Comment:
This function is not returning the fiat value.

It returns the total amount of token units considering all chains